src/data_validation/validation.py

```diff
@@ -0,0 +1,73 @@
+import os
+import toml
+
+from src.utils.hdfs_mods import read_hdfs_csv as read_data
+
+datafilepath = "/ons/rdbe_dev/Frozen_Group_Data2021_244_Headers.csv"
+
+
+def load_schema(filePath: str = "./config/DataSchema.toml") -> dict:
+    """Load the data schema from toml file into a dictionary
+
+    Keyword Arguments:
+        filePath -- Path to data schema toml file
+        (default: {"./config/DataSchema.toml"})
+
+    Returns:
+        A dict: dictionary containing parsed schema toml file
+    """
+    file_exists = os.path.exists(filePath)
+
+    # Check if DataSchema.toml exists
+    if not file_exists:
+        return file_exists
+    else:
+        # Load toml data schema into dictionary
+        toml_string = toml.load(filePath)
+    return toml_string
+
+
+def check_data_shape(
+    dataFile: str = datafilepath,
+    filePath: str = "./config/DataSchema.toml",
+    numCols: int = 93,
+) -> bool:
+    """Compares the shape of the data and compares it to the shape of the toml
+    file based off the data schema. Returns true if there is a match and false
+    otherwise.
+
+    Keyword Arguments:
+        dataFile -- Path to data file to compare (default: {datafilepath})
```

Comment:
Refer to Python naming conventions. Camel case is not normal for Python here

Reply:
Amended now to lower case with underscores separating words